.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,58 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```

Comment:
Do i understand correctly: Only on creation of new releases the publish is triggered?